src/renderer/views/Playlist/Playlist.js

```diff
@@ -14,7 +14,7 @@ import {
   getLocalPlaylistContinuation,
   parseLocalPlaylistVideo,
 } from '../../helpers/api/local'
-import { extractNumberFromString, setPublishedTimestampsInvidious, showToast } from '../../helpers/utils'
+import { extractNumberFromString, getIconForSortPreference, setPublishedTimestampsInvidious, showToast } from '../../helpers/utils'
```

Comment:
You got 4 items here maybe make it multiline...
```suggestion
import {
  extractNumberFromString,
  getIconForSortPreference,
  setPublishedTimestampsInvidious,
  showToast,
} from '../../helpers/utils'
```

Reply:
This seems like something we should have linter rules for